tests/vertexonly/test_interpolation_from_parent.py

```diff
@@ -133,6 +131,13 @@ def test_scalar_function_interpolation(parentmesh, vertexcoords, fs):
     vm = VertexOnlyMesh(parentmesh, vertexcoords)
     vertexcoords = vm.coordinates.dat.data_ro.reshape(-1, parentmesh.geometric_dimension())
     fs_fam, fs_deg, fs_typ = fs
+    if (
+        parentmesh.coordinates.function_space().ufl_element().family()
+        == "Discontinuous Lagrange"
+        and fs_fam == "CG"
+    ):
+        pytest.skip(f"Interpolating into f{fs_fam} on a periodic mesh is not well-defined.")
```

Comment:
feels like the frontend interpolation should raise a `ValueError` so that users don't get this ill-definedness.

Reply:
It's a little more complex than that though. If one takes a function which is actually periodic and does the interpolation then it's all fine, and that's an obvious use case. It's just in general that the interpolation isn't defined.